src/DeepCopy/Matcher/Doctrine/DoctrineProxyMatcher.php

```diff
@@ -3,7 +3,7 @@
 namespace DeepCopy\Matcher\Doctrine;
 
 use DeepCopy\Matcher\Matcher;
-use Doctrine\Common\Persistence\Proxy;
+use Doctrine\Persistence\Proxy;
```

Comment:
Doctrine already published a forward compatibility alias, so we don't need to care about it:
https://github.com/doctrine/persistence/blob/1.3.8/lib/Doctrine/Common/Persistence/Proxy.php